lib/rom/rails/railtie.rb

```diff
@@ -79,9 +79,12 @@ def setup
       end
 
       def infer_default_repository
-        return unless defined?(ActiveRecord)
-        spec = ActiveRecord::Configuration.call
-        [:sql, spec[:uri], spec[:options]]
+        if RepositoryConfiguration.inferrable?
+          spec = RepositoryConfiguration.infer
+          [:sql, spec[:uri], spec[:options]]
+        elsif ENV['DATABASE_URL']
+          [:sql, ENV['DATABASE_URL']]
```

Comment:
I'm going to hold off on the relocation for the moment.  I think we need to decide how many helper classes we need beforehand.

If we're having a separate helper for each source we can infer the configuration from, then this probably deserves its own (simple) one.  Currently it's not a good fit for the `RepositoryConfiguration` class which is specifically dealing with options hashes.